src/include/btree_inline.h

```diff
@@ -691,6 +734,10 @@ __wt_cache_page_evict(WT_SESSION_IMPL *session, WT_PAGE *page)
           session, &btree->bytes_updates, modify->bytes_updates, "WT_BTREE.bytes_updates");
         __wt_cache_decr_check_uint64(
           session, &cache->bytes_updates, modify->bytes_updates, "WT_CACHE.bytes_updates");
+        __wt_cache_decr_check_uint64(session, &btree->bytes_delta_updates,
+          modify->bytes_delta_updates, "WT_BTREE.bytes_delta_updates");
+        __wt_cache_decr_check_uint64(session, &cache->bytes_delta_updates,
+          modify->bytes_delta_updates, "WT_CACHE.bytes_delta_updates");
```

Comment:
This function doesn't make changes to any of the `WT_PAGE_MODIFY.*` stats. Do those live elsewhere? If so, should that location also be updated?

Reply:
Is there stats for page modify?

Reply:
At the page level, they are being evicted. No need to update them.

Reply:
> Are there stats for page modify?

Yes, the one being incremented in `__wt_cache_page_inmem_decr_delta_updates`.

> At the page level, they are being evicted. No need to update them.

Oh, I see now. Thanks for explaining that.